bing.com.BingWebMasterDPOO.json

```diff
@@ -0,0 +1,22 @@
+{
+  "providerId": "bing.com",
+  "providerName": "Microsoft Bing",
+  "serviceId": "BingWebMasterDPOO",
+  "serviceName": "Domain proof of ownership with Bing Webmaster services",
+  "version": 2,
+  "logoUrl": "https://domainconnect.org/wp-content/uploads/<<bing logo>> link",
+  "description": "Verifies domain ownership for further configuration of Microsoft Bing Services",
+  "variableDescription": "Guid of webmaster for verification of domain ownership with Bing webmaster enables quick and easy SEO on Bing.",
+  "syncBlock": false,
+  "shared": false,
+  "warnPhishing": false,
+  "syncRedirectDomain": "www.bing.com",
```

Comment:
Are you sure you want syncRedirectDomain to contain www?  This means any redirect url passed in would need to have a domain of the form www.bing.com.  bing.com (or foo.bing.com) wouldn't work.

Most templates ommit the www.

Reply:
yes, their redirect url is with www

Reply:
But why would they want to limit this?  By putting in www.bing.com, the redirect can only be to www.bing.com.  If you put in bing.com, you could redirect to www.bing.com or foo.bing.com. Doesn't matter much at the end of the day I guess.

Reply:
AFAIK we don't have such language in the spec to match also subdomains for syncRedirectDomain.
Assuming that more specific is more secure for redirect_urls, it's reasonable to keep the exact match requirement for syncRedirectDomain. In case more hosts are needed, it is comma-separated list for this purpose.